README.md

```diff
@@ -11,7 +11,11 @@ With this app, you will be able to download translation patches for your Jackbox
 
 ## Features
 
-- Supports the following languages: <img src="https://raw.githubusercontent.com/lipis/flag-icons/0df85234c05404587baf0a7cb2c4e82c36ed4648/flags/1x1/fr.svg" width="12" height="12" /> French, <img src="https://raw.githubusercontent.com/lipis/flag-icons/0df85234c05404587baf0a7cb2c4e82c36ed4648/flags/1x1/de.svg" width="12" height="12" /> German, <img src="https://raw.githubusercontent.com/lipis/flag-icons/0df85234c05404587baf0a7cb2c4e82c36ed4648/flags/1x1/es.svg" width="12" height="12" /> Spanish, <img src="https://raw.githubusercontent.com/lipis/flag-icons/0df85234c05404587baf0a7cb2c4e82c36ed4648/flags/1x1/ua.svg" width="12" height="12" /> Ukrainian
+- Supports the following languages: 
+<img src="https://raw.githubusercontent.com/lipis/flag-icons/0df85234c05404587baf0a7cb2c4e82c36ed4648/flags/1x1/fr.svg" width="12" height="12" /> French, 
+<img src="https://raw.githubusercontent.com/lipis/flag-icons/0df85234c05404587baf0a7cb2c4e82c36ed4648/flags/1x1/de.svg" width="12" height="12" /> German, 
+<img src="https://raw.githubusercontent.com/lipis/flag-icons/0df85234c05404587baf0a7cb2c4e82c36ed4648/flags/1x1/es.svg" width="12" height="12" /> Spanish, 
+<img src="https://raw.githubusercontent.com/lipis/flag-icons/0df85234c05404587baf0a7cb2c4e82c36ed4648/flags/1x1/ua.svg" width="12" height="12" /> Ukrainian
```

Comment:
When merging with the upstream/dev branch to resolve the merge conflicts, I also made the change to put every flag with its country in one line. This helps with readability when editing the file. And you don't have to search for the text between of dozen of links. 
> *This is more some thinking for the future, if even more languages get supported and there for more entries of countries will be added*